tests/test_dataset.py

```diff
@@ -24,9 +24,6 @@
 
 import git
 import pytest
-import yaml
-
-from renku.models.datasets import Creator, Dataset, DatasetFile
```

Comment:
Why remove? This is needed for `test_creator_parse`.

Reply:
oops